content/departments/cloud/technical-docs/v1.1/mi1-1_enable_executors_process.md

```diff
@@ -155,3 +139,13 @@ sourcegraph-executors-docker-registry-mirror  us-central1-a  n1-standard-2
 ```
 
 If above all check out, visit the [Compute Engine Console](https://console.cloud.google.com/compute/instances) and check logs of the executor instance for more troubleshooting.
+
+### Troubleshooting executor-dependent features
+
+To check if executors are making progress on available work, consult this [alert](https://docs.sourcegraph.com/admin/observability/alerts#executor-executor-processor-total).
+
+Executor-dependent features (Server Side Batch Changes, Code-Intelligence Auto-indexing) differ in the way they use executors & influence core Sourcegraph experience, for further troubleshooting contact:
+
+- #wg-shipping-executors for general executor questions
+- #code-intel for Code Intelligence auto-indexing questions, [on-call](https://sourcegraph.app.opsgenie.com/teams/dashboard/d0c10593-3edd-4d7e-8d1b-2ad29afeaa71/main)
+- #batch-changes for Server-Side Batch Changes questions, [on-call](TODO)
```

Comment:
we have slack handle `@batcher-support` if that suffices for now?

Reply:
Ideally, there should be an OpsGenie rotation supporting SSBC (so we can route alerts correctly) - not sure if that's something Engineers or EMs ( cc @chrispine @RafLeszczynski ) configure.

Reply:
Added it to todays team sync doc.

Reply:
With the distribution of our team and the various hours we work, we actually have pretty good coverage already. And to my knowledge, we've never had any issues with taking too long to respond or missing an SLA. So in the spirit of "If it ain't broke, don't fix it," I would strongly prefer that we continue to use `@batcher-support` until that feels like it is no longer working.

Reply:
We don't have a formal SLA / on-call policy for Cloud, so if you think Slack based support is enough to provide acceptable customer experience, let's leave this without OpsGenie for now.